File: CTDK_Python/001/_001.py
from math import sqrt

print("Diem thu nhat: ")
x1 = float(input("Nhap hoanh do: "))
y1 = float(input("Nhap tung do: "))

print("Diem thu hai: ")
x2 = float(input("Nhap hoanh do: "))
y2 = float(input("Nhap tung do: "))

d = sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1))

print("Diem thu nhat: ")
print("Hoanh do: ", x1)
print("Tung do: ", y1)

print("Diem thu nhat: ")
print("Hoanh do: ", x2)
print("Tung do: ", y2)

print("Khoang cach: ", d)

File: LTHAM/001/001.cpp
#include<iostream>
#include<cmath>
using namespace std;

void Nhap(float&, float&); // hoanh do va tung do
float KhoangCach(float, float, float, float); // tinh khoang cach
void Xuat(float, float);// xuat hoanh do va tung do

int main()
{
	float x1, y1;
	Nhap(x1, y1);
	float x2, y2;
	Nhap(x2, y2);
	cout << "Diem thu nhat: " << endl;
	Xuat(x1, y1);
	cout << "Diem thu hai: " << endl;
	Xuat(x2, y2);
	float kq = KhoangCach(x1, y1, x2, y2);
	cout << "Khoang cach la: " << kq << endl;
	return 0;
}

void Nhap(float& xx, float& yy)
{
	cout << "Nhap hoanh do: " << endl;
	cin >> xx;
	cout << "Nhap tung do: " << endl;
	cin >> yy;
}

void Xuat(float xx, float yy)
{

	cout << "Hoanh do: " << xx << endl;
	cout << "Tung do: " << yy << endl;
}

float KhoangCach(float x1, float y1, float x2, float y2)
{
	return sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));
}